projects/plugins/jetpack/extensions/shared/components/inspector-notice/index.jsx

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
Maybe not in this PR, but I think we could move this to the new `@automattic/jetpack-shared-extension-utils` in a follow-up PR.

Reply:
Thank you for telling us where to put it! 👍

Reply:
I see TC builds are failing. Gonna have to resolve this.
```
[11:59:49 ] ERROR in ./extensions/shared/components/inspector-notice/index.jsx 6:0-61
[11:59:49 ] Module not found: Error: Can't resolve '@automattic/jetpack-components'
```

Reply:
This should now be fixed as of D75088-code and D75094-code.

Reply:
Thank you very very much!